src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -7,11 +7,15 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>19.2.0-preview</Version>
+		<Version>20.0.0-preview</Version>
```

Comment:
@MikhailTryakhov any word on when you are planning to release a stable version here? Any reason why you plan to do a major version bump here

Reply:
@dsgouda  I think we need publish stable version when all know issues will be fixed. now we have one in brooklyn team. let's I test all swagger issues we have (in optional jobs for example), fix them and next Ignite version will be stable

Reply:
if you are in preview you cannot bump up major version